pkg/master/master.go

```diff
@@ -707,9 +712,10 @@ func (m *Master) getExtensionResources(c *Config) map[string]rest.Storage {
 		deploymentStorage := deploymentetcd.NewStorage(restOptions("deployments"))
 		storage["deployments"] = deploymentStorage.Deployment
 		storage["deployments/status"] = deploymentStorage.Status
-		// TODO(madhusudancs): Install scale when Scale group issues are fixed (see issue #18528).
-		// storage["deployments/scale"] = deploymentStorage.Scale
 		storage["deployments/rollback"] = deploymentStorage.Rollback
+		if !m.ApiGroupVersionOverrides["autoscaling/v1"].Disable {
```

Comment:
That, I think, will be problematic for two reasons:

1. We call Creater.New(gvk) to get the versionedObject for decoding in api_installer.go. Creater.New(gvk) which really is Scheme.New(gvk) checks if gvk is registered before creating the object, fails otherwise. This is causing the tests to fail now. We could probably override this behavior by always creating the versionedObject for subresources from SubresourceGroupVersion override. But then
2. We will be returning objects whose type will be registered nowhere.